content/about/index.md

```diff
@@ -45,24 +47,24 @@ We'll be having an election for SVNA officers in July. If you like hanging out w
 
 Email: <board@santiagovilla.org>
 
-Our current President is Bee Hanson, one of the original founders of the SVRA, and steadfast recorder of all SVNA and Mountain View Mobile Home Association (MVMHA) activities. Bee has lived in the neighborhood since 1996, and is looking forward to many more years here.
+Our current President is Bee Hanson, who documents all SVNA and MVMHA (Mountain View Mobile Home Association) activities. Bee has lived in the neighborhood since 1996, and is looking forward to many more years here.
 
 ### SVNA's Nonprofit Status
 
-In 2020, Trey Bornmann set up SVNA as a nonprofit \[501 (c) (3)], which means our association is now a service organization. Trey also wrote the initial version of the association bylaws. You can read the latest update here.
-
-Bee Hanson wrote the Letter of Certification that describes the SVNA's mission as a service organization, and Trey provided most of the legal documentation.
+In 2020, Trey Bornmann set up SVNA as a nonprofit \[501 (c) (3)], which means our association is now a service organization. Trey provided the original legal documentation, and Bee updated it this year.
 
-Soon, we will be able to solicit donations from organizations whose interests in North Bayshore coincide with our own. But that's something we won't be doing until we have a specific objective, and that might be far in the future.
+Nonprofit status gives us the ability to solicit donations from organizations whose interests in North Bayshore coincide with our own. This gives us some flexibility to respond to future financial opportunities that might come our way.
 
 ### Our Financial Status
 
-In 2017, SVNA qualified to receive an annual neighborhood association support grant from the City of Mountain View in the amount of $1800. We used those funds to get our association started with a range of neighborhood activities. The annual support grant was increased the following year to $2000. Because of the impact on the City of the COVID-19 pandemic, the current grant gradually decreased, and is now $904 annually.
+In 2017, SVNA qualified to receive an annual neighborhood association support grant from the City of Mountain View in the amount of $1800. We used those funds to get our association started. The annual support grant was increased the following year to $2000, but because of the impact on the City of the COVID-19 pandemic, and the proliferation of new neighborhood associations, the current grant gradually decreased, and is now $904 annually. Board members have contributed when we want to try something new, and residents have often contributed baked goods and tasty dishes to our social events.
```

Comment:
```suggestion
Since 2017, SVNA has qualified to receive an annual neighborhood association support grant from the City of Mountain View, originally in the amount of $1800. We used those funds to get our association started. The annual support grant was increased the following year to $2000, but because of the impact on the City of the COVID-19 pandemic, and the proliferation of new neighborhood associations, the current grant gradually decreased, and is now $904 annually. Board members have contributed when we want to try something new, and residents have often contributed baked goods and tasty dishes to our social events.
```